crates/sui-sdk/Cargo.toml

```diff
@@ -18,8 +18,7 @@ signature = "1.6.0"
 tokio = "1.20.1"
 rand = "0.8.5"
 bcs = "0.1.3"
-
-bip39 = { git = "https://github.com/patrickkuo/rust-bip39.git" , rev = "a76fe8310416555e6383b42b8acc4eb93c7bcc89", features = ["rand"]}
```

Comment:
i dont see why we need a fork for this - i also prefer tiny-bip39 as it is a more popular and updated version for rust-bip39

<img width="931" alt="image" src="https://user-images.githubusercontent.com/108701016/189168395-817d64b5-668a-4ea1-9d95-7c0f58f4b879.png">

Reply:
the reason for the fork is to workaround the unicode-normalization deps version clash, feel free to replace it!